index.html

```diff
@@ -23,6 +23,7 @@
 </head>
 <body>
```

Comment:
- [ ] The only thing left you to need to do is to add the JavaScript linter in the linters.yml file. This is a mandatory request because you are working already with JavaScript. [Here is the link ](https://github.com/microverseinc/linters-config/tree/master/html-css-js)again.